examples/smooth-params.js

```diff
@@ -0,0 +1,126 @@
+goog.require('ol.Attribution');
+goog.require('ol.Map');
+goog.require('ol.View');
+goog.require('ol.control');
+goog.require('ol.control.Attribution');
+goog.require('ol.control.Zoom');
+goog.require('ol.extent');
+goog.require('ol.layer.Tile');
+goog.require('ol.proj');
+goog.require('ol.source.OSM');
+goog.require('ol.source.WMTS');
+goog.require('ol.tilegrid.WMTS');
+
+
+// create the WMTS tile grid in the google projection
+var projection = ol.proj.get('EPSG:3857');
+var tileSizePixels = 256;
+var tileSizeMtrs = ol.extent.getWidth(projection.getExtent()) / tileSizePixels;
+var matrixIds = [];
+var resolutions = [];
+var attributions = [
+  new ol.Attribution({
+    html: '<a href="http://scalgo.com">SCALGO</a>'
+  }),
+  new ol.Attribution({
+    html: '<a href="http://www.cgiar-csi.org/data/' +
+        'srtm-90m-digital-elevation-database-v4-1">CGIAR-CSI SRTM</a>'
+  })
+];
+
+for (var i = 0; i <= 14; i++) {
+  matrixIds[i] = i;
+  resolutions[i] = tileSizeMtrs / Math.pow(2, i);
+}
+var tileGrid = new ol.tilegrid.WMTS({
+  origin: ol.extent.getTopLeft(projection.getExtent()),
+  resolutions: resolutions,
+  matrixIds: matrixIds
+});
+
+var scalgoToken = 'CC5BF28A7D96B320C7DFBFD1236B5BEB';
+
+// In this source the "threshold" dimension is specified as a "dynamic"
+// dimension. This means that when updateDimensions is called to change the
+// value of this dimension the "old" tiles are not invalidated, instead they're
+// used as "interim" tiles while the new tiles are loading.
+var dynamicSource = new ol.source.WMTS({
+  url: 'http://ts2.scalgo.com/global/wmts?token=' + scalgoToken,
+  layer: 'hydrosheds:sea-levels',
+  format: 'image/png',
+  matrixSet: 'EPSG:3857',
+  attributions: attributions,
+  tileGrid: tileGrid,
+  style: 'default',
+  dimensions: {
+    'threshold': 100
+  },
+  dynamicDimensions: ['threshold']
```

Comment:
It may be clear already, but "dynamic dimensions" are dimensions that the user will change (with a time slider for example), and for which we will use interim tiles while new tiles are loading. But I agree with you when you say that the term doesn't really convey what this does. I just don't have a better name right now.